### DISCUSSION TITLE
Tracking setup

### DISCUSSION THREAD

**Discussion by devdems** [ZoneMinder/zoneminder#3830] [Q&A]
How can i setup tracking for my camera https://www.galayou-store.com/y4. It has PTZ and I'm able to move the camera with zoneminder but it does not track when detection is made.

This is my setup:

![image](https://github.com/ZoneMinder/zoneminder/assets/8215353/ba6e3ef2-a58c-410e-9a4f-dc86d5edd061)

**Comment by connortechnology** [comment]
Tracking may require a certain type of PTZ support. Some cameras support Start Moving/Stop Moving commands.  The tracking I think requires the ability to say Move to this spot.  Check your logs there are likely messages about this.

It may be possible to get it working just need to figure out how long to move in that direction to get to where we want to be.
